src/vstest.console/Processors/EnableLoggerArgumentProcessor.cs

```diff
@@ -199,6 +243,18 @@ private static void HandleInvalidArgument(string argument)
                     argument));
         }
 
+        /// <summary>
```

Comment:
In general, prefer InheritDoc instead of generic docs

Reply:
done